core/src/mindustry/ui/fragments/BlockInventoryFragment.java

```diff
@@ -198,17 +198,6 @@ public void touchUp(InputEvent event, float x, float y, int pointer, KeyCode but
         }
     }
 
-    private String round(float f){
```

Comment:
This is not equivalent to the standard UI formatting; it doesn't display decimal thousands, for example. Have you tested how this look in inventories with large amounts of items?